integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/CoupleMBusDevice.feature

```diff
@@ -518,7 +518,7 @@ Feature: SmartMetering Installation - Couple M-Bus Device
       | GatewayDeviceIdentification    | <GatewayDeviceIdentification> |
       | Channel                        |                             1 |
     When the Couple G-meter "TESTG101205673117" request is received for E-meter "TEST1024000000001" with force
-    Then the Couple response is "OK"
+    Then the Couple response has "TESTG101205673117", channel "1" and PrimaryAddress "3"
```

Comment:
Waarom is deze step niet geparameteriseerd vanuit de Examples? waarom ook niet de step 'a dlms device' net als de simulator

Reply:
Dat is inderdaad netter, maar ik bedoelde eigenlijk dat de waarden van de parameters ook in de Examples tabel staan gespecificeerd.

Reply:
Aaaaah, dat kan niet voor dit example...... Deze test koppelt niet altijd de gateway vanuit de examples

Reply:
check